README.md

```diff
@@ -23,14 +27,14 @@ Prior to commit, run:
 ```shell
 pre-commit run --all-files
 ```
-
+-----
```

Comment:
This is more of a stylistic thing, and I Am Not A Designer, but I don't find the horizontal lines that useful on the README. In my opinion, the headings do enough to provide clean section breaks.

Reply:
Does look better now thanks